lib/model.js

```diff
@@ -81,7 +80,7 @@ assign(props, {
   ns: { type: 'string', default: undefined },
   isSrvRecord: { type: 'boolean', default: false },
   hostname: { type: 'string', default: 'localhost' },
-  port: { type: 'port', default: 27017 },
```

Comment:
there should be somewhere a custom datatype .. not sure where, that at this point we probably don't use anymore

Reply:
Not sure I understand – this PR is removing the custom data type (because that’s what the ticket says should be done)?

Reply:
The 'port' datatype should not be something native in ampersand, i think we wrote some code to make it happen, probably we can drop that code too.

Reply:
@mcasimir This PR is about dropping that code – the custom data type was defined in `lib/data-types.js`, which is now deleted?

Reply:
Oh the whole file was deleted 🤦 🤦 🤦 🤦, ok thanks!